src/zh/zazhimi/src/eu/kanade/tachiyomi/extension/zh/zazhimi/Response.kt

```diff
@@ -0,0 +1,70 @@
+package eu.kanade.tachiyomi.extension.zh.zazhimi
+
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SManga
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class IndexResponse(
+    val status: String,
+    val error: String,
+    val new: List<NewItem>,
+)
+
+@Serializable
+data class ShowResponse(
+    val status: String,
+    val error: String,
+    val content: List<ShowItem>,
+)
+
+@Serializable
+data class SearchResponse(
+    val status: String,
+    val error: String,
+    val magazine: List<SearchItem>,
+)
+
+@Serializable
+data class NewItem(
+    val magId: String,
+    val magName: String,
+    val magCover: String,
+    val magDate: String,
+) {
+    fun toSManga(): SManga = SManga.create().apply {
+        title = this@NewItem.magName
+        author = this@NewItem.magName.split(" ")[0]
+        thumbnail_url = this@NewItem.magCover
+        url = "/show.php?a=${this@NewItem.magId}"
+        initialized = true
+    }
+}
+
+@Serializable
+data class ShowItem(
+    val magId: String,
+    val magName: String,
+    val typeId: String,
+    val typeName: String,
+    val cateId: String,
+    val magPic: String,
+    val pageUrl: String,
+    val pageThumbUrl: String,
+) {
+    fun toPage(i: Int): Page = Page(i, this@ShowItem.magPic, this@ShowItem.magPic)
```

Comment:
Assuming `url` is not used:
```suggestion
    fun toPage(i: Int): Page = Page(i, imageUrl = this@ShowItem.magPic)
```